static/css/style.css

```diff
@@ -1684,22 +1683,44 @@ input.input-has-error {
 }
 
 .banner-logo {
-  background-position: center center;
+  background-position: top center;
   background-size: contain;
-  height: 72px;
-  width: 72px;
+  height: 60px;
+  width: 60px;
+  min-height: 60px;
+  min-width: 60px;
   background-repeat: no-repeat;
-  margin: 0 .25em auto 1em;
+  margin: 0 1.25rem 0 2.5rem;
 }
 
 .banner-relay-logo {
+  background: none;
+  position: relative;
+}
+
+.banner-relay-logo::after {
+  content: "";
+  width: 70px;
+  height: 60px;
+  position: absolute;
+  left: 0;
+  top: 0;
+  display: block;
   background-image: url("/static/images/banner-addon.svg");
+  background-repeat: no-repeat;
+  background-position: top center;
+  background-size: contain;
 }
 
 .banner-fx-logo {
   background-image: url("/static/images/logos/fx-logo.svg");
 }
 
+/* TODO: Replace MPP logo */
```

Comment:
I'm adding a TODO anywhere the temporary assets are located, to make updating before launch easier.